README.md

```diff
@@ -31,7 +31,7 @@ There are two ways to get a source tree:
   Note: cloning GHC from Github requires a special setup. See [Getting a GHC
   repository from Github] [7].
 
-  **DO NOT submit pull request directly to the github repo.**
+  We will accept easily-reviewed documentation changes as GitHub Pull Requests.
```

Comment:
"easily-reviewed" makes sense but I don't think that PRs should be limited to documentation changes up front.

[The proposal](https://github.com/ghc-proposals/ghc-proposals/pull/11) certainly wasn't limited to documentation changes only.